packages/core/src/operations.ts

```diff
@@ -322,6 +332,40 @@ export class TxOperations implements Omit<Client, 'notify'> {
     }
     return doc
   }
+
+  private splitMixinUpdate<T extends Doc>(
+    update: DocumentUpdate<T>,
+    mixClass: Ref<Class<T>>
+  ): {
+      docUpdate: DocumentUpdate<T>
+      mixinUpdate: DocumentUpdate<T>
+    } {
+    const docUpdate: DocumentUpdate<T> = {}
+    const mixinUpdate: DocumentUpdate<T> = {}
+
+    const mixinAttrs = this.getHierarchy().getOwnAttributes(mixClass)
```

Comment:
Please use getAllAttributes and split based on attribute of value.

Reply:
thanks, fixed